common/models/data-source-definition.js

```diff
@@ -18,8 +18,17 @@ module.exports = function(DataSourceDefinition) {
       }
       const connector = DataSourceDefinition.getConnector();
       const id = data.id;
-      //TODO(Deepak) - add response handling later as part of the callback
+      // TODO(Deepak) - add response handling later as part of the callback
```

Comment:
I still think you should leave the TODO notes outside the source code and in your own notes. Just delete the lines altogether and fix it when the time comes.